### PR TITLE
User/vlvesic/cnn workshop updates

### PR DESCRIPTION
**Pull request by VesS0**
Updating mostly comments of the workshop

### CODE REVIEW

File: workshops/conv_nets/runner.py [hunk 1]
```diff
@@ -43,6 +43,12 @@ def get_feed_dict(self, image):
         :returns: Dictionary that maps input nodes to data.
             See [`tf.Session.run`](https://www.tensorflow.org/api_docs/python/tf/Session#run).
         """
+
+        # There are multiple ways to alter dimensions of image, in order to expland dimensions you can use:
```
Comment (VesS0) [inline workshops/conv_nets/runner.py]: Not image, but tensor

File: workshops/conv_nets/README.md [hunk 2]
```diff
@@ -28,6 +28,19 @@ General comments:
 * Some task may require you to implement many methods. However, implementations are very short -- up to
 10 lines of code, usually far fewer.
 
+Before you start it is important to examine project a bit. Navigate to conv_nets.py file to the main function at the bottom.
+You will see how executing this file with certain number will do one of the following things:
+ 1. Prepare dataset, which makes sure all of the images are present on the machine and shows you how they look.
+ 2. Evaluate image, which runs random image from prepared dataset through the model, predicting and visualizing the class it belongs to.
+ 3. Evaluate dataset, which calculates loss and accuracy of the model (how well is model performing) on test set.
```
Comment (mnidza) [inline workshops/conv_nets/README.md]: Nit: is model performing -> the model performs

File: workshops/conv_nets/README.md [hunk 3]
```diff
@@ -28,6 +28,19 @@ General comments:
...
+ 4. 6. Train, which will train the model in order to improve its performance (Calculated in step 3) and store trained weights on the disk.
```
Comment (mnidza) [inline workshops/conv_nets/README.md]: Nit: Calculated -> calculated

File: workshops/conv_nets/README.md [hunk 4]
```diff
@@ -28,6 +28,19 @@ General comments:
...
+ 5. 7. Evaluate image, which is identical to the one in number 2, but is using trained model stored on the disk.
+
+In simple words, the relation between `model.py`, `runner.py` and `trainer.py` files can be seen as following:
+`model.py` contains the definition of the network, and its nodes.
+`runner.py` is used to produce prediction on a signle image, based on the given model (deifned in `model.py`).
```
Comment (mnidza) [inline workshops/conv_nets/README.md]: signle -> single

File: workshops/conv_nets/README.md [hunk 5]
```diff
@@ -70,26 +83,30 @@ TensorFlow provides other high-level APIs, such as
...
+Note that these function are called from `conv_nets.evaluate_image` method in `conv_nets.py`.
 
 A network is executed by creating a `tf.Session` object and calling its
 [`run` method](https://www.tensorflow.org/api_docs/python/tf/Session#run).
 A session object encapsulates the control and state of the TensorFlow runtime.
 The `run` method requires a set of outputs to be computed, and a mapping of input placeholders to actual data items.
 
 In `runner.py` implement the following methods in `Runner` class:
-* `get_fetches`, which defines which network nodes should be evaluated.
+* `get_fetches`, which defines which network nodes(outputs) should be evaluated.
```
Comment (mnidza) [inline workshops/conv_nets/README.md]: Nit: space before 'outputs'

File: workshops/conv_nets/runner.py [hunk 6]
```diff
@@ -43,6 +43,12 @@ def get_feed_dict(self, image):
         :returns: Dictionary that maps input nodes to data.
             See [`tf.Session.run`](https://www.tensorflow.org/api_docs/python/tf/Session#run).
         """
+
+        # There are multiple ways to alter dimensions of array, in order to expland dimensions you can use:
```
Comment (mnidza) [inline workshops/conv_nets/runner.py]: expland -> expand

**Pull request merged**